### ISSUE TITLE
Aspect ratios are a little off

### ISSUE THREAD

**Issue by CloudDweller** [OpenEmu/OpenEmu#204]
I've been doing some testing with the Snes9x core and it appears that the aspect ratios are a little off.  For example, below is a screen grab from the native Snes9x app running windowed at native resolution.

Snes9x - Windowed
<a href="http://tinypic.com?ref=wpuut" target="_blank"><img src="http://i49.tinypic.com/wpuut.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

Here Snes9x is drawing the game at 256 x 239 with the black bars top and bottom but if we remove the black bars then the resolution drops to 256 x 224 which is the native resolution for the game (in this case Super Mario World).

Now running that same game with OpenEmu's Snes9x core distorts the game producing a slightly narrower horizontal resolution.  Its hard to see this as OpenEmu doesn't allow you to run the game at the native resolution.  Here you can see a comparison between the two at full screen.

Snes9x - Full Screen
<a href="http://tinypic.com?ref=2qw0c2u" target="_blank"><img src="http://i50.tinypic.com/2qw0c2u.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

OpenEmu - Full Screen
<a href="http://tinypic.com?ref=2qak1ht" target="_blank"><img src="http://i45.tinypic.com/2qak1ht.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

Using Photoshop to scale the full screen image from Snes9x down to a width of 256 produces a height of 224 which is correct.  Doing the same to OpenEmu again using a width of 256, produces a height of 239 which is wrong.

Its also worth noting that although the SNES hardware's native resolution for this game is 256 x 224, the nature of CRT TV's caused the image to stretch horizontally as the TV's aspect ratio was 4:3 and not 8:7 which is what the SNES hardware used.  This can be seen in the image below.

Snes9x - 4:3 Full Screen
<a href="http://tinypic.com?ref=2806zbb" target="_blank"><img src="http://i48.tinypic.com/2806zbb.png" border="0" alt="Image and video hosting by TinyPic"></a>

So to sum up.
1. OpenEmu is not drawing the game at the native resolution and therefor the correct aspect ratio which I think needs fixing.
2. I think that there should be an option for a 4:3 aspect ratio so that the game is drawn at the same aspect ratio as when it was played on a CRT TV using real hardware.
